SSA/Experimental/Bits/Fast/BitStream.lean

```diff
@@ -418,8 +418,48 @@ theorem not_congr (e1 : a ≈ʷ b) : (~~~a) ≈ʷ ~~~b := by
   intros g h
   simp only [not_eq, e1 g h]
 
+theorem ofBitVec_not' : ofBitVec (~~~ x) ≈ʷ ~~~ ofBitVec x := by
+  intros n a
+  simp only [ofBitVec, a, ↓reduceIte, BitVec.getLsb_not, decide_True, Bool.true_and, not_eq]
```

Comment:
A terminal simp does not need to have `simp only`. This might allow you to reduce the list of explicit theorems.